src/main/java/core/basesyntax/services/impl/ShopServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.services.impl;
```

Comment:
The package declaration should not be followed by an empty line. Please remove the empty line after the package declaration.